src/klein/resource.py

```diff
@@ -59,5 +59,5 @@ def __repr__(self):
 
 
 preserve = modules[__name__]
-modules[__name__] = _SpecialModuleObject()
-modules[__name__].__preserve__ = preserve
+modules[__name__] = _SpecialModuleObject()  # type: ignore
```

Comment:
Post this change, I think I have an idea for a slightly less gross way to express this.